src/filters/mask/mask.wgsl

```diff
@@ -79,20 +77,22 @@ fn mainFragment(
 
     var maskClamp = filterUniforms.uMaskClamp;
 
-     var clip = step(3.5,
-        step(maskClamp.x, filterUv.x) +
-        step(maskClamp.y, filterUv.y) +
-        step(filterUv.x, maskClamp.z) +
-        step(filterUv.y, maskClamp.w));
+    var clip = step(3.5,
```

Comment:
clip is unused?

Reply:
Oops, I mistakenly removed it from shader.. will return it back

Reply:
Fixed